packages/AutoTDD-GUI.package/ATDDGitHubActionResults.class/instance/buildSimplifySmalltalkCiLogCheckBoxWith..st

```diff
@@ -0,0 +1,9 @@
+building
+buildSimplifySmalltalkCiLogCheckBoxWith: aBuilder
+
+	^ aBuilder pluggableCheckBoxSpec new
+		model: self;
+		label: #simplifySmalltalkCiLogCheckBoxLabel;
+		action: #simplifySmalltalkCiLogCheckBoxAction;
+		state: #simplifySmalltalkCiLogCheckBoxState;
+		yourself
```

Comment:
```suggestion
building
buildSimplifyLogCheckBoxWith: aBuilder

	^ aBuilder pluggableCheckBoxSpec new
		model: self;
		label: #simplifyLogCheckBoxLabel;
		action: #simplifyLogCheckBoxAction;
		state: #simplifyLogCheckBoxState;
		yourself
```
I felt the names were a little too long to support readability. Although the parsing is in the same class i would not include any info on implementation (SmalltalkCiLog vs Log) in GUI code